lock.rst

```diff
@@ -193,6 +194,11 @@ To lock the default resource, autowire the lock using
     same lock, inject the ``LockFactory`` instead to create a separate lock
     instance for each service.
 
+    .. deprecated:: 5.2
+
+        The ``LockInterface`` service and ``lock`` alias are deprecated since
+        Symfony 5.2.
+
```

Comment:
I don't think this is needed, as we don't document it anymore. (and Symfony's deprecation system already does a good job educating people about deprecations & the required changes)

Reply:
REMOVED